src/SegmentationCameraSensor.cc

```diff
@@ -18,32 +18,34 @@
 #include <mutex>
 #include <memory>
 
-#include "ignition/common/Console.hh"
-#include "ignition/common/Profiler.hh"
-#include "ignition/common/Image.hh"
-#include "ignition/msgs.hh"
-#include "ignition/rendering/SegmentationCamera.hh"
+#include <ignition/common/Console.hh>
+#include <ignition/common/Profiler.hh>
+#include <ignition/common/Image.hh>
```

Comment:
```suggestion
#include <ignition/common/Console.hh>
#include <ignition/common/Image.hh>
#include <ignition/common/Profiler.hh>
```
nit: alphabetical order